packages/core/utils/src/modules-sdk/joiner-config-builder.ts

```diff
@@ -70,6 +70,10 @@ export function defineJoinerConfig(
 
     while (!stopSearching) {
       const fullPath = getCallerFilePath(index)
+      if (!fullPath) {
+        stopSearching = true
```

Comment:
suggestion: Should we maybe just do `while (true)` and remove the `stopSearching` variable (and use `break` instead)? I think it might clean up the code if we have only 1 way of exiting the loop

Reply:
yes you are right 👍 let me change that